L63-core-call-credentials-debug-string.md

```diff
@@ -0,0 +1,121 @@
+Title
+----
+* Author(s): mhaidry
+* Approver: a11r
+* Status: Draft
+* Implemented in: cpp
+* Last updated: [Date]
+* Discussion at: <google group thread> (filled after thread exists)
```

Comment:
I have some feedback about the design beyond the typographical suggestions I've posted to GitHub. Is there a discussion thread about this yet?

Reply:
Sure, I'll follow up with a google group. The doc is not quite ready for review yet as this is a draft PR. Once the doc is ready I'll post a group as well. Thanks for the formatting comments.

Reply:
Done

Reply:
Hi @chwarr please feel free to provide any feedback in the discussion link. Thanks for the review.

Reply:
I don't actually have any. :-) When I was composing it, I realized it was wrong.